opm/common/OpmLog/LogBackend.hpp

```diff
@@ -75,6 +79,7 @@ namespace Opm
         int64_t m_mask;
         std::shared_ptr<MessageFormatterInterface> m_formatter;
         std::shared_ptr<MessageLimiter> m_limiter;
+        int64_t m_level;
```

Comment:
> Currently LogBackend::m_mask does not have any functionality, does it?

Well - I would say it does. 

1. When creating a `LogBackend` instance the base constructor should be called to set the `m_mask` flag: see e.g. https://github.com/OPM/opm-common/blob/master/opm/common/OpmLog/StreamLog.cpp#L25

2. When adding a message to a backend the `LogBackend::includeMessage()` method is consulted - and that is based on the `m_mask`: https://github.com/OPM/opm-common/blob/master/opm/common/OpmLog/LogBackend.cpp#L58